packages/dd-trace/src/encode/tags-processors.js

```diff
@@ -41,8 +41,6 @@ function truncateToLength (value, maxLength) {
 function truncateSpan (span) {
   return fromEntries(Object.entries(span).map(([key, value]) => {
     switch (key) {
-      case 'resource':
-        return ['resource', truncateToLength(value, MAX_RESOURCE_NAME_LENGTH)]
```

Comment:
Stats still need the 5,000 char limit which is why `MAX_RESOURCE_NAME_LENGTH` is still exported and isn't deleted entirely.